src/sysinfo.cc

```diff
@@ -389,6 +389,8 @@ std::string GetSystemName() {
 #else // defined(BENCHMARK_OS_WINDOWS)
 #ifdef BENCHMARK_OS_MACOSX //Mac Doesnt have HOST_NAME_MAX defined
 #define HOST_NAME_MAX 64
+#elif defined(BENCHMARK_OS_QNX)
+#define HOST_NAME_MAX 154
```

Comment:
i think this is defined in limits.h for qnx (http://www.qnx.co.uk/developers/docs/6.5.0/index.jsp?topic=%2Fcom.qnx.doc.neutrino_prog%2Fposix_conformance.html)

Reply:
Here is what in QNX SDP 7:

```
>>grep -r "HOST_NAME_MAX" ./usr/include
./usr/include/confname.h:#define _SC_HOST_NAME_MAX					154
./usr/include/limits.h:#undef	_POSIX_HOST_NAME_MAX
./usr/include/limits.h:#define	_POSIX_HOST_NAME_MAX	255
```

Reply:
thanks for checking!